### DISCUSSION TITLE
Remix.run landing-page code

### DISCUSSION THREAD

**Discussion by zvictor** [remix-run/remix#1741] [Q&A]
Hi everyone! I am new here (coming from years of nextjs dev) and I gotta ask it right away.

**Who has built the [remix.run](https://remix.run) landing page?? It's glorious! ❤️🚀**

I am craving for it's source code to be open to the public so I can learn a thing or two about awesomeness, and hopefully contribute to that page one day :sparkles::dizzy:

**Is it publicly available somewhere? If not, can we make it open? 😊**

🚀💎

![image](https://user-images.githubusercontent.com/181076/151826325-a315539d-9532-4064-ae7f-9aa0390e81da.png)

**Comment by carlopascual** [comment]
Same here! Looks super awesome 🤩

**Comment by erikfiala** [comment]
This would be amazing 😍
